src/MainDriver.java

```diff
@@ -20,5 +26,30 @@ public static void main(String[] args) {
         } else {
             System.out.print("Sorry, I didn't get that. Re-run the program to try again!");
         }
+
+        scanner.close();
+    }
+
+    public static double getNumericInput() { // a method to obtain a numeric from the user
+        double userValue = 0.0;
+        try {
+            System.out.print("Please enter a number to convert from USD to CAD: ");
+            Scanner scn = new Scanner(System.in);
+            userValue = scn.nextDouble();
+            scn.close();
+        } catch (InputMismatchException e) { //in the case that a value other than an int or a double is entered
+                System.out.println("That was not a valid numeric. Please rerun the program to try again");
+                System.exit(-1); //just ending the system if this happens at the moment. Can change implementation if needed.
+            }
```

Comment:
Minor: make sure curly bracket indentation is correct: it needs to be closed at the same vertical point where you closed the last one, in this case.